pkg/lib/admin/authz/audit.go

```diff
@@ -0,0 +1,29 @@
+package authz
+
+import "context"
+
+type auditContextKeyType struct{}
```

Comment:
Should be auth context, instead of audit context. Rename the JSON key as well.

Reply:
@louischan-oursky I think name it as `AuthContext` looks like something which store auth state / session etc, but in fact we only use it as audit. The content inside is also not verified. People can claim himself as any user.

Reply:
Let's stick with `audit_context` then.